src/server/registry/referrers.go

```diff
@@ -0,0 +1,170 @@
+package registry
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/go-openapi/swag"
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+
+	"github.com/goharbor/harbor/src/lib/errors"
+	lib_http "github.com/goharbor/harbor/src/lib/http"
+	"github.com/goharbor/harbor/src/lib/q"
+	"github.com/goharbor/harbor/src/pkg/accessory"
+	"github.com/goharbor/harbor/src/pkg/artifact"
+	"github.com/goharbor/harbor/src/server/router"
+	"github.com/goharbor/harbor/src/server/v2.0/handler"
+)
+
+const ReferrersSchemaVersion = 2
+const ReferrersMediaType = "application/vnd.oci.image.index.v1+json"
+
+func newReferrersHandler() http.Handler {
+	return &referrersHandler{}
```

Comment:
Don't need to initialize the two fields(artifactManager/accessoryManager) in the structure?

Reply:
updated